tests/testthat/test_messages.R

```diff
@@ -10,5 +10,12 @@ test_that("Invalid input variable returns error", {
     expect_error(setvar(core, NA, "global.permafrost.beta", 10, ""),
                  regexp = "Invalid input variable: '")
     shutdown(core)
+})
+
+test_that("Default fetchvar without dates returns error", {
```

Comment:
Could we also test a nondefault variable like a halocarbon RF or some other random time series variable? The way that the test is set up and described gives the impression that this error will only occur for the default fetchvar variables.

Reply:
Well, that _is_ what the error will catch: default variables (`vars` parameter is NA) and no dates (NA).

Reply:
Haha okay true. So only three variables are defined as default variables 
`default_fetchvars <- c(ATMOSPHERIC_CO2, RF_TOTAL, RF_CO2, GLOBAL_TEMP)`

I guess my question is why are we treating these variables differently than other Hector variables? Or was the behavior that problematic, the return values when date == NA only any issue for these three variables?

Reply:
My thought was that if someone does
```r
fetchvars(hc, years = NA, vars = GLOBAL_TEMP())
```
they'll get an error but then can look at the documentation and understand that global temperature requires dates. But if they do
```r
fetchvars(hc)
```
they might be confused because they haven't requested (they don't think) any variables.